src/fasta-recs.py

```diff
@@ -11,8 +10,22 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # print(f"Now I need to process the records in {args.fasta}")
+
+    printer = ''
+    for line in (args.fasta):
```

Comment:
You don't need the parentheses around `args.fasta` here. There is nothing wrong with it, it is just not what the typical Python programmer would expect.